src/schema/sources.ts

```diff
@@ -273,6 +274,16 @@ export const typeDefs = /* GraphQL */ `
       """
       token: String
     ): Source! @auth
+
+    """
+    Deletes a inactive squad
+    """
+    removeSource(
```

Comment:
Remove sounds less strict than delete

```suggestion
    deleteSource(
```